pkg/xen-tools/init.sh

```diff
@@ -29,6 +29,7 @@ if [ -d /proc/xen/ ]; then
 
 elif [ -e /dev/kvm ]; then
    echo "KVM hypervisor support detected"
+   while true ; do sleep 60 ; done
```

Comment:
Are there zero daemons which we should watchdog for KVM?

Reply:
There will be -- but not yet ;-) 

However, we need it alive to be able to eve exec into it.